### PR TITLE
i18n fixes for confirmChangePopup

### PR DESCRIPTION
**Pull request by rzats**
### Contains

Minor fixes to code added in #2574 to make UI strings properly internationalizable and a note in the wiki about translating your stuff. Also fixes an unrelated typo I happened to encounter in player settings.

### How to test

Check that the strings added in #2574 (confirmChangePopup) are properly translated.

### Outstanding before merging

Nothing! Pinging @skaldarnar and @zeokav for review.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/resources/assets/i18n/menu.lang [hunk 1]
```diff
@@ -50,6 +50,8 @@
     "category-mouse": "category-mouse",
     "category-movement": "category-movement",
     "category-nui": "category-nui",
+    "change-popup-message": "change-popup-message",
```
Comment (skaldarnar) [inline engine/src/main/resources/assets/i18n/menu.lang]: I'd go for a more descriptive key `change-keybind-popup-message` as "change" is quite general and could conflict with other change actions in the menu later. 
Another option would be move it to another .lang file specific for keybindings....

Comment (rzats) [inline engine/src/main/resources/assets/i18n/menu.lang]: Noted!

I'd rather not overcomplicate things re: Weblate setup unless we're going for a fully reworked system with strings split into 5-6 files.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**